packages/react-aria-components/src/Checkbox.tsx

```diff
@@ -220,13 +220,14 @@ function Checkbox(props: CheckboxProps, ref: ForwardedRef<HTMLLabelElement>) {
       isRequired: props.isRequired || false
     }
   });
+  let {focusableProps} = useFocusable(props, ref);
```

Comment:
I think this is giong to double up focus events, they should already supported on Checkbox through `useCheckbox->useToggle`?